packages/itwin/reports-config-widget/src/widget/context/ReportsApiConfigContext.tsx

```diff
@@ -6,14 +6,20 @@ import type { AccessToken } from "@itwin/core-bentley";
 import * as React from "react";
 import { createContext } from "react";
 
+export type GetAccessTokenFn = () => Promise<AccessToken>;
+
 export interface ReportsApiConfig {
-  getAccessToken: () => Promise<AccessToken>;
+  getAccessToken: GetAccessTokenFn;
+  iTwinId: string;
+  iModelId: string;
```

Comment:
Why is iModelId required for ReportsAPI?

Reply:
I've renamed the context to `ReportsConfigContext` for broader generalization. Additionally, I removed `iModelId` since it was only utilized in one instance: during the addition of Mappings to the report. Previously, the `iModelId` was used to default-select the list of iModels from which to choose mappings.